### PR TITLE
support cluster scale up via sealer apply cli

### PR DESCRIPTION
**Pull request by kakaZhou719**

### CODE REVIEW

**Comment by VinceCui** [comment]
@kakaZhou719 Resolve conflict.

File: cmd/sealer/cmd/cluster/apply.go [hunk 1]
```diff
@@ -58,94 +71,239 @@ will apply the diff change of current Clusterfile and the original one.`,
...
-			//save desired clusterfile
-			if err = cf.SaveAll(); err != nil {
-				return err
-			}
 
-			cluster := cf.GetCluster()
-			infraDriver, err := infradriver.NewInfraDriver(&cluster)
-			if err != nil {
-				return err
+			if !osi.IsFileExist(common.DefaultKubeConfigFile()) {
```
Comment (VinceCui) [inline cmd/sealer/cmd/cluster/apply.go:75]: You can check if k8s exists here, more security.

**Review by VinceCui — COMMENTED** [review]
Very nice work~

File: cmd/sealer/cmd/cluster/apply.go (hunk 1 above)
Comment (kakaZhou719) [reply cmd/sealer/cmd/cluster/apply.go:75]: init k8s client to check ?

**Review by VinceCui — APPROVED** [review]
LGTM

**Pull request merged**